plugin-tester-scala/src/main/scala/example/myapp/helloworld/GreeterClient.scala

```diff
@@ -56,7 +66,7 @@ object GreeterClient {
       val responseStream = client.itKeepsReplying(HelloRequest("Alice"))
       val done: Future[Done] =
         responseStream.runForeach(reply => println(s"got streaming reply: ${reply.message}"))
-      Await.ready(done, 1.minute)
+      Await.ready(done, 1.minute) // just to keep sample simple - dont do Await.ready in actual code
```

Comment:
Perhaps we should avoid it here as well then - but let's keep that for another PR